docs/deployer.md

```diff
@@ -1,23 +1,36 @@
 # Deployer
 
-Deployer is a class that helps you deploy [Algorand Standard Assets(ASA)](https://developer.algorand.org/docs/features/asa/) and [Stateful Smart Contracts(SSC)](https://developer.algorand.org/docs/features/asc1/stateful/) to the Algorand network.
+Deployer wraps an SDK `AlgodV2` client and provides a higher level functionality to deploy [Algorand Standard Assets(ASA)](https://developer.algorand.org/docs/features/asa/) and [Stateful Smart Contracts(SSC)](https://developer.algorand.org/docs/features/asc1/stateful/) to the Algorand network:
+* load ASA definition files
+* load smart-contract files
+* create transaction log and checkpoints.
 
-You can deploy a ASA from the `sample-project` with the deploy script `scripts/0-sampleScript.js`,
+It will protect you to deploy same ASA or steteful smart contract twice. It will store transaction log in a checkpoint and allow you to reference later (in other scripts or in REPL) deployed ASA.
```

Comment:
```suggestion
It will protect you from deploying same ASA or stateful smart contract twice. It will store transaction log in a checkpoint and allow you to reference later (in other scripts or in REPL) deployed ASA.
```

Reply:
Good suggestion. Thanks.